src/main/scala/com/itechart/ny_accidents/utils/ParserUtils.scala

```diff
@@ -0,0 +1,57 @@
+package com.itechart.ny_accidents.utils
+
+import java.time.LocalDateTime
+
+import com.itechart.ny_accidents.constants.GeneralConstants
+import com.itechart.ny_accidents.constants.GeneralConstants.ZERO_DATE
+import org.apache.spark.sql.Row
+import java.sql.{Date => SqlDate}
+
+import scala.util.Try
+
+object ParserUtils {
+
+  def toLongFromInt(accident: Row, column: Int): Option[Long] = {
+    Try(accident.getInt(column).toLong).filter(value => value != 0).toOption
+  }
+
+  def toLocalDate(dateStr: String, timeStr: String): LocalDateTime = {
+    val dateTimeStr = dateStr + " " + timeStr
+    DateUtils.parseDate(dateTimeStr, GeneralConstants.DATE_TIME_ACCIDENTS_FORMATTER).getOrElse(ZERO_DATE)
+  }
+
+  def toMillis(row: Row, dateColumn: Int, timeColumn: Int): Option[Long] = {
+    val dateTime = row.getString(dateColumn) + " " + row.getString(timeColumn)
+    val value = DateUtils.parseDateToMillis(dateTime, GeneralConstants.DATE_TIME_ACCIDENTS_FORMATTER).getOrElse(0L)
+    if (value == 0) None
```

Comment:
Try to remove if else here and in other methods of class

Reply:
b727113986a6495d83c6b8bc2c161dccc516818e